src/gui/Console.cpp

```diff
@@ -0,0 +1,529 @@
+/*                        C O N S O L E . C P P
+ * BRL-CAD
+ *
+ * Copyright (c) 2025 United States Government as represented by
+ * the U.S. Army Research Laboratory.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public License
+ * version 2.1 as published by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this file; see the file named COPYING for more
+ * information.
+ */
+/** @file Console.cpp */
+
+
+#include <QAbstractItemView>
+#include <QApplication>
+#include <QClipboard>
+#include <QCompleter>
+#include <QKeyEvent>
+#include <QMimeData>
+#include <QPlainTextEdit>
+#include <QRegularExpression>
+#include <QScrollBar>
+#include <QStringListModel>
+#include <brlcad/CommandString/CommandString.h>
+#include <brlcad/Database/Database.h>
```

Comment:
This include is not necessary, because it was already included by CommandString.h.